example/FamilyHubs.Example/Pages/Examples/FullPages/SingleAutocomplete/SingleAutocomplete.cshtml.cs

```diff
@@ -0,0 +1,55 @@
+using FamilyHubs.SharedKernel.Razor.ErrorNext;
+using FamilyHubs.SharedKernel.Razor.FullPages.SingleAutocomplete;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using System.Collections.Immutable;
+
+namespace FamilyHubs.Example.Pages.Examples.FullPages.SingleAutocomplete;
+
+public record Dto(long Id, string Name);
+
+public class SingleAutocompleteModel : PageModel, ISingleAutocompletePageModel
```

Comment:
queastion: the implementation of this looks almost identical to `SingleAutocomplete-Custom`. Is there a way to easily combine these two in some way? No worries if not!

Reply:
Just noticed this is in the `FamilyHubs.Example` project, so I'm guessing you're just providing two implementations of the same component as an example. If so, just ignore this!

Reply:
Yeah, just 2 different examples, each as stripped back, simple and easy to follow as possible.